composer.json

```diff
@@ -41,6 +41,7 @@
         "laminas/laminas-stdlib": "3.18.0",
         "laminas/laminas-xmlrpc": "2.17.0",
         "lcobucci/jwt": "4.2.1",
+        "league/csv": "9.8.0",
```

Comment:
Any reason to bring in yet another csv handler? 

There is native fgetcsv/fputcsv for pure csv files. In addition phpspreadsheet can handle xlsx, ods files in addition to csv.